float64 north
float64 east
float32 depth
float32 velocity_depth
bool mission_enable
bool depth_only
uint16 waypoint_number
uint64  wall_time
uint64  time_to_next_waypoint
